integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/LoggingExporter.java

```diff
@@ -3,10 +3,6 @@
 
 package oracle.weblogic.kubernetes.actions.impl;
```

Comment:
Modify the Copyright Statement to include year 2023

Reply:
Changed